sdk/trace/sampling.go

```diff
@@ -47,11 +47,11 @@ const (
 	// Drop will not record the span and all attributes/events will be dropped.
 	Drop SamplingDecision = iota
 
-	// RecordOnly indicates the span's `IsRecording() == true`, but `Sampled` flag
+	// RecordOnly indicates the span's IsRecording method returns true, but [trace.FlagsSampled] flag
```

Comment:
`[ ]` does not work for fields.

Also `* *` should be removed.

See 
![image](https://github.com/user-attachments/assets/b5ff9e10-202b-4834-87fa-a6cdaa859799)

Reply:
> [ ] does not work in this context.

Yeah...though it works on Jetbrain's IDE.

> Also * * should be removed.

Good catch

Reply:
> though it works on Jetbrain's IDE.

I also know that other things related to Go Doc Comments work in Jetbrain's IDE that render differently on pkg.go.dev but is also interpreted differently by the Go tooling. E.g. for Go tooling `// Deprecated:` must be after a "blank" comment line: `//` .